File: examples/benchmarks/oo6.py
class Foo:
    def __init__(self):
	self.i = 10
	self.j = 20
    def I(self, i):
	self.i = i
    def J(self, j):
	self.j = j

class main:
    o = Foo()
    for i in range(1,500001):
	o.I(i)
	o.J(i)
    print o.i

File: examples/benchmarks/oo6.rb
#! ruby

class Foo
	def initialize()
		@i = 10
		@j = 20
	end
	def i
		@i
	end
	def i=(i)
		@i = i
	end
	def j=(j)
		@j = j
	end
end

o = Foo.new
for i in 1..500000
  o.i = i
  o.j = i
end
puts o.i


